Dockerfile-alpine.template

```diff
@@ -52,11 +52,19 @@ RUN set -ex \
 		tk-dev \
 		xz-dev \
 		zlib-dev \
+                libffi-dev \
+                expat-dev \
 # add build deps before removing fetch deps in case there's overlap
 	&& apk del .fetch-deps \
 	\
 	&& cd /usr/src/python \
+        && rm -r Modules/expat \
+		Modules/zlib \
+		Modules/_ctypes/darwin* \
+		Modules/_ctypes/libffi* \
 	&& ./configure \
+		--with-system-ffi \
+                --with-system-expat \
```

Comment:
The previous version of Python 3.x seemed to have worked without similar segmentation faults. But I've just run the regression tests in the Python 3.4 container with the settings above and they behaved the same way as without. So it does seem to work either way. I haven't tested all 3 versions, though. 

Do you want me to push updated Dockerfiles generated with the `update.sh` script and let them run on CI to test?

It would also be possible to use the docker `--build-arg` setting for running the unittest as part of the build process locally. For instance, adding this to the Alpine Dockerfiles:
```
...
ARG RUN_TESTS
...
	&& make -j$(getconf _NPROCESSORS_ONLN) \
        && if [ $RUN_TESTS ]; then LD_LIBRARY_PATH=/usr/src/python ./python -m test.regrtest; fi \
	&& make install \
...
```

And then running the docker command with `RUN_TESTS=1` would also run the tests without modifying any of the Docker images hosted on the Hub:

```
docker build --build-arg RUN_TESTS=1 --tag tested-python 3.4/alpine
```

Would that be something that makes sense to add for testing purposes? What do you suggest as the next steps?